src/features/import/l10n/messageIds.ts

```diff
@@ -122,6 +122,20 @@ export default makeMessages('feat.import', {
       tags: m('Tags'),
       zetkinHeader: m('Zetkin'),
     },
+    preview: {
+      columnHeader: {
+        ext: m('External ID'),
+        int: m('ID'),
```

Comment:
The name of this string (`int`) confuses me. Doesn't a string for this already exist in `globalMessageIds`? Otherwise, if we keep it here, I still think it should just be called `id`.